EIPS/eip-executable-proposal

```diff
@@ -0,0 +1,79 @@
+---
+eip: <to be assigned>
+title: Executable Proposal Standard
+description: A standard for interfaces of executable proposals that can operate smart contract
+author: Zainan Victor Zhou (xinbenlv)<zzn@zzn.im>
+discussions-to: TBC
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): ERC
+created: 2022-07-13
+requires (*optional): 7
+---
+
+## Abstract
+A standrized format for proposals that will be executable. 
```

Comment:
Should be longer. This is perhaps _too_ trite.

Reply:
Updated with more details